pyproject.toml

```diff
@@ -28,6 +28,7 @@ tomlkit = "*"
 typing-extensions = "*"
 platformdirs = "^4.3"
 lxml = "*"
+distro = "*"
```

Comment:
I don't think such simple checks justifies a dependency. 

Just use `platform`, you can call out to `lsb_release -d` if you want specific distro.

Reply:
Will do!

Reply:
@ErikBjare My system does not have `lsb_release -d` installed on default expecting more devices probably don't have installed by default. A workaround suggested is to read the contents of `/etc/os-release`